### ISSUE TITLE
loss NAN

### ISSUE THREAD

**Issue by ledashuai** [rosinality/glow-pytorch#37]
In my experiment, when the data set is small, the experiment can run normally. When the number of data sets is large, the loss value always changes to Nan when there are thousands of epochs

**Comment by rosinality** [comment]
Glow training is not very stable. You can adjust training or model hyperparameters like batch sizes.
